yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -1532,19 +1557,44 @@ func addMigrationCaveatsToAssessmentReport(unsupportedDataTypesForLiveMigration
 		if len(unsupportedDataTypesForLiveMigrationWithFForFB) > 0 {
 			columns := make([]ObjectInfo, 0)
 			for _, colInfo := range unsupportedDataTypesForLiveMigrationWithFForFB {
-				columns = append(columns, ObjectInfo{ObjectName: fmt.Sprintf("%s.%s.%s (%s)", colInfo.SchemaName, colInfo.TableName, colInfo.ColumnName, colInfo.DataType)})
-
-				assessmentReport.AppendIssues(AssessmentIssue{
-					Category:            MIGRATION_CAVEATS_CATEGORY,
-					CategoryDescription: GetCategoryDescription(MIGRATION_CAVEATS_CATEGORY),
-					Type:                "UNSUPPORTED_DATATYPE_LIVE_MIGRATION_WITH_FF_FB",
-					Name:                queryissue.UNSUPPORTED_DATATYPE_LIVE_MIGRATION_WITH_FF_FB_ISSUE_NAME,
-					Impact:              constants.IMPACT_LEVEL_1,
-					Description:         UNSUPPORTED_DATATYPES_FOR_LIVE_MIGRATION_WITH_FF_FB_DESCRIPTION,
-					ObjectType:          constants.COLUMN,
-					ObjectName:          fmt.Sprintf("%s.%s.%s (%s)", colInfo.SchemaName, colInfo.TableName, colInfo.ColumnName, colInfo.DataType), // TODO (fix): adding datatype here is temporary fix
-					DocsLink:            UNSUPPORTED_DATATYPE_LIVE_MIGRATION_DOC_LINK,
-				})
+				qualifiedColName := fmt.Sprintf("%s.%s.%s", colInfo.SchemaName, colInfo.TableName, colInfo.ColumnName)
+				columns = append(columns, ObjectInfo{ObjectName: fmt.Sprintf("%s (%s)", qualifiedColName, colInfo.DataType)})
+
+				var datatype string
+				if strings.Contains(colInfo.DataType, ".") {
+					datatype = strings.Split(colInfo.DataType, ".")[1]
+				} else {
+					datatype = colInfo.DataType
+				}
+
+				var queryIssue queryissue.QueryIssue
+
+				if colInfo.IsArrayType && colInfo.IsEnumType {
+					queryIssue = queryissue.NewArrayOfEnumDatatypeIssue(
```

Comment:
are these issue - array and enum types part of the list you are maintaining for all unsupported issues etc.